### DISCUSSION TITLE
Do explicit optimizePackageImports override or merge with the defaults?

### DISCUSSION THREAD

**Discussion by mmkal** [vercel/next.js#62007] [Help]
### Summary

There's a short list of libraries documented which are optimized by default: https://nextjs.org/docs/app/api-reference/next-config-js/optimizePackageImports

If we set the list explicitly, will those in that list still be optimized, or should we copy over the list?

**Comment by capibawa** [comment]
They would be merged. https://github.com/vercel/next.js/blob/canary/packages/next/src/server/config.ts#L1013
